.github/workflows/wheel.yaml

```diff
@@ -0,0 +1,38 @@
+name: Wheels
+
+on: [push, pull_request]
+
+jobs:
+  build_wheels:
+    name: Build wheel on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-18.04, windows-latest, macos-latest]
```

Comment:
Shouldn't this be CentOS 6 for `manylinux2010` compatibility?

ref: https://www.python.org/dev/peps/pep-0571/

Reply:
No. ubuntu is only host for manylinux docker container (quay.io/pypa/manylinux2010_x86_64 and quay.io/pypa/manylinux2010_i686)